templates/ts-scrape-single-page/package.json

```diff
@@ -1,20 +1,25 @@
 {
-    "name": "getting-started-typescript",
+    "name": "ts-scrape-single-page",
     "version": "0.0.1",
     "type": "module",
     "description": "This is an example of an Apify actor.",
+    "engines": {
+        "node": ">=18.0.0"
+    },
     "dependencies": {
-        "apify": "^3.0.0"
+        "apify": "^3.1.4",
+        "axios": "^1.4.0",
+        "cheerio": "^1.0.0-rc.12"
     },
     "devDependencies": {
         "@apify/tsconfig": "^0.1.0",
         "ts-node": "^10.9.1",
-        "typescript": "^4.9.5"
+        "typescript": "^5.0.4"
```

Comment:
We might want to add `@types/node` to all the TS templates, IIRC some mapped types are not working correctly without that. But I guess that can be a separate PR narrowing it in all the templates.